src/chains.ts

```diff
@@ -139,19 +143,20 @@ export const ETHEREUM_MAINNET: Chain = {
   name: 'Ethereum',
   symbol: 'ETH',
   mainnet: true,
-  evm: true,
   logo: 'https://raw.githubusercontent.com/pangolindex/sdk/master/src/images/chains/eth.png',
-  pangolin_is_live: true,
+  pangolin_is_live: false,
```

Comment:
we need to keep `pangolin_is_live` true as we are showing this chain in Network selection modal. please revert this change below also everywhere